mixer/adapter/inventory.go

```diff
@@ -15,15 +15,10 @@
 package adapter
```

Comment:
do we need this file at all now?

Reply:
In the next PR, I will delete this. Removing it now has cascading effects which I was trying to avoid in this PR